package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Social media platform sentiment listening service",
   "main": "index.js",
   "scripts": {
-    "start:dev": "nodemon --inspect=0.0.0.0 ./app/server.ts",
+    "start:dev": "ts-node --files ./app/server.ts",
```

Comment:
Check if integration of `nodemon` with `ts-node` is possible while using `--files` flag.
`files` flag is needed to accept `include` property from `tsconfig.json` so as to include custom `@types` for npm packages that don't have them.